api/provenance/provenance_test.go

```diff
@@ -49,12 +49,60 @@ func TestProvenance_Semver(t *testing.T) {
 
 func mockModule(version string) debug.Module {
 	return debug.Module{
-		Path:    "sigs.k8s.io/kustomize/kustomize/v5",
+		Path:    provenance.ModulePath,
 		Version: version,
 		Replace: nil,
 	}
 }
 
+func mockBuildInfo(mainVersion, depsVersion string) *debug.BuildInfo {
```

Comment:
Hi @koba1t 
Could you please review it.
Thanks in advance

Reply:
Hi @koba1t 
Any chance to get it reviewed and merged?
Thanks in advance,

Reply:
Sorry for delay response.
I'll check.